docs/features/features.rst

```diff
@@ -21,6 +21,7 @@ It includes:
 - Spatially heterogeneous spectra (see :ref:`see line-of-sight <label_los_index>`)
 - Post-processing tools to load and :ref:`compare with experimental spectra <label_spectrum_howto_compare>` (see :ref:`the Spectrum object <label_spectrum>`)
 - A :ref:`Line Survey <label_spectrum_linesurvey>` tool to identify which lines correspond to a spectral feature.
+- Matlab users can access RADIS easily and use it from Matalb `https://github.com/radis/radis/tree/develop/examples`
```

Comment:
Here you can directly insert a link to the example page, using the `:ref:label`.. Syntax above and adding the same label as a header in your example.
Warning, as far as I remember there is a trailing underscore `_` difference between the label and its references, look up Sphinx documentation for exact syntax

Reply:
I have done the changes sir. If there is anything else which i need to do let me know